addons/im_livechat/models/mail_channel.py

```diff
@@ -105,7 +105,14 @@ def _channel_get_livechat_partner_name(self):
         if self.livechat_operator_id in self.channel_partner_ids:
             partners = self.channel_partner_ids - self.livechat_operator_id
             if partners:
-                return ', '.join(partners.mapped('name'))
+                partner_name = False
+                for partner in partners:
+                    if not partner_name:
+                        partner_name = partner.name
+                    else:
+                        partner_name += ', %s' % partner.name
+                    partner_name += ' (%s)' % partner.country_id.name if partner.country_id else ''
```

Comment:
Is it possible to add a test for this?

Reply:
I would use a `if` for better readability, like:

```
if partner.country_id:
    partner_name += ' (%s)' % parter.country_id.name

Reply:
Done, would you please review it again?
I've added the test in https://github.com/odoo/odoo/pull/33831.
Thanks